diff --git a/common/css/common.css b/common/css/common.css
index c93b029..5998732 100644
--- a/common/css/common.css
+++ b/common/css/common.css
@@ -48,9 +48,7 @@ a:hover {
     text-decoration: none;
 }
 
-.btn {
-    padding: 1px;
-}
+.btn {}
 
 .btn:focus {
     outline: none !important;
@@ -85,12 +83,10 @@ a:hover {
 }
 
 .btn-circle.btn-lg {
-    text-align: center;
-    width: 50px;
-    height: 50px;
-    padding: 9px;
-    border-radius: 25px;
-}
+    padding: 5%;
+    border-radius: 50%;
+    -webkit-box-shadow: 0 0 5px rgba(0, 0, 0, .125);
+            box-shadow: 0 0 5px rgba(0, 0, 0, .125);}
 
 .btn-icon {
     width: 16px;
@@ -105,10 +101,6 @@ a:hover {
     padding-right: 0px;
 }
 
-header {
-    text-align: center;
-}
-
 #title,
 #baseline {
     display: inline;
@@ -130,16 +122,11 @@ header {
     font-size: 16px;
 }
 
-#visualizer-row {
-    margin-left: 0px;
-    margin-right: 0px;
-    margin-bottom: 1px;
+.custom-row {
+    margin-bottom: 10px;
 }
 
-#visualizer-row > div {
-    padding-left: 1px;
-    padding-right: 1px;
-}
+#visualizer-row > div {}
 
 #visualizer .bar,
 #visualizer .beat-label {
@@ -173,7 +160,6 @@ header {
     font-weight: bold;
     font-family: sans-serif;
     font-size: 13px;
-    text-align: center;
     vertical-align: top;
 }
 
@@ -186,43 +172,11 @@ header {
     color: lightgray;
 }
 
-#controls-row {
-    margin-left: 0px;
-    margin-right: 0px;
-}
-
-#controls-row > div {
-    padding-left: 0px;
-    padding-right: 0px;
-}
-
-#left-col,
-#right-col {
-    text-align: center;
-    padding-left: 2px;
-    padding-right: 2px;
-}
-
-#left-col > .row,
-#right-col > .row {
-    margin-left: 0px;
-    margin-right: 0px;
-}
-
-#right-col label.label {
-    background-color: #999;
-    display: block;
-    padding: 1px;
-    margin-bottom: 17px;
-}
+#controls-row {}
 
-#palo-and-options {
-    padding-left: 0px;
-    padding-right: 0px;
-}
+#controls-row > div {}
 
 #palo {
-    margin-bottom: 2px;
     cursor: pointer;
 }
 
@@ -267,17 +221,12 @@ header {
 #tempo-label,
 #volume-label {
     margin-bottom: 17px;
+    font-size: 85%;
 }
 
-#tempo-slider-container {
-    padding-left: 1px;
-    padding-right: 0.5px;
-}
+#tempo-slider-container {}
 
-#volume-slider-container {
-    padding-left: 0.5px;
-    padding-right: 1px;
-}
+#volume-slider-container {}
 
 .slider.slider-vertical {
     height: 144px;
@@ -319,7 +268,6 @@ header {
 footer {
     border-top: 1px solid #e5e5e5;
     padding: 20px 0;
-    text-align: center;
     margin-top: 10px;
 }
 
@@ -346,14 +294,10 @@ footer {
         font-size: 30px;
     }
 
-    .btn {
-        padding: 3px 5px;
-    }
+    .btn {}
 
     #palo,
-    .btn-group {
-        margin-bottom: 10px;
-    }
+    .btn-group {}
 
    .btn-icon {
         width: 32px;
@@ -369,18 +313,9 @@ footer {
 @media all and (min-width: 992px) and (orientation: landscape) {
 
     #palo,
-    .btn-group {
-        margin-bottom: 30px;
-    }
-
-    .btn {
-        padding: 5px 10px;
-    }
+    .btn-group {}
 
-    header {
-        padding-top: 10px;
-        padding-bottom: 10px;
-    }
+    .btn {}
 
     #title {
         font-size: 80px;
@@ -390,8 +325,7 @@ footer {
         font-size: 45px;
     }
 
-    #visualizer-row {
-        margin-top: 20px;
+    .custom-row {
         margin-bottom: 20px;
     }
 
@@ -414,10 +348,6 @@ footer {
         display: block;
     }
 
-    #right-col label.label {
-        padding: 5px;
-    }
-
     .slider.slider-vertical {
         height: 250px;
     }
@@ -427,15 +357,12 @@ footer {
 /* 1200px is the limit width from medium to large with bootstrap */
 @media all and (min-width: 1200px) and (orientation: landscape) {
 
-    #visualizer-row {
-        margin-top: 50px;
+    .custom-row {
         margin-bottom: 50px;
     }
 
     #palo,
-    .btn-group {
-        margin-bottom: 45px;
-    }
+    .btn-group {}
 
 }
 
@@ -443,12 +370,6 @@ footer {
 
 @media all and (min-height: 480px) and (orientation: portrait) {
 
-    header {
-        padding-top: 5px;
-        padding-bottom: 5px;
-        text-align: center;
-    }
-
     #title,
     #baseline {
         display: block;
@@ -463,18 +384,13 @@ footer {
         margin-bottom: 10px;
     }
 
-    .btn {
-        padding: 2px 3px;
-    }
+    .btn {}
 
-    #visualizer-row {
-        margin-top: 15px;
+    .custom-row {
         margin-bottom: 15px;
     }
 
-    #palo {
-        margin-bottom: 10px;
-    }
+    #palo {}
 
     .slider.slider-vertical {
         height: 200px;
@@ -489,13 +405,7 @@ footer {
         padding-right: 10px;
     }
 
-    header {
-        padding-top: 10px;
-        padding-bottom: 10px;
-    }
-
-    #visualizer-row {
-        margin-top: 30px;
+    .custom-row {
         margin-bottom: 30px;
     }
 
@@ -504,9 +414,7 @@ footer {
     }
 
     #palo,
-    .btn-group {
-        margin-bottom: 20px;
-    }
+    .btn-group {}
 
 }
 
@@ -526,13 +434,7 @@ footer {
         font-size: 30px;
     }
 
-    header {
-        padding-top: 20px;
-        padding-bottom: 20px;
-    }
-
-    #visualizer-row {
-        margin-top: 45px;
+    .custom-row {
         margin-bottom: 45px;
     }
 
@@ -568,13 +470,7 @@ footer {
         padding-right: 20px;
     }
 
-    header {
-        padding-top: 30px;
-        padding-bottom: 30px;
-    }
-
-    #visualizer-row {
-        margin-top: 60px;
+    .custom-row {
         margin-bottom: 60px;
     }
 
@@ -591,9 +487,7 @@ footer {
     }
 
     #palo,
-    .btn-group {
-        margin-bottom: 30px;
-    }
+    .btn-group {}
 
     .play {
         width: 100px;
@@ -613,11 +507,6 @@ footer {
         padding-right: 25px;
     }
 
-    header {
-        padding-top: 40px;
-        padding-bottom: 40px;
-    }
-
     #title {
         font-size: 80px;
     }
@@ -626,15 +515,12 @@ footer {
         font-size: 45px;
     }
 
-    #visualizer-row {
-        margin-top: 80px;
+    .custom-row {
         margin-bottom: 80px;
     }
 
     #palo,
-    .btn-group {
-        margin-bottom: 40px;
-    }
+    .btn-group {}
 
     .slider.slider-vertical {
         height: 450px;
diff --git a/common/js/common.js b/common/js/common.js
index a3a2d92..9d447c8 100644
--- a/common/js/common.js
+++ b/common/js/common.js
@@ -1182,10 +1182,10 @@ function setPalo(paloSlug) {
         value: paloData.defaultTempo,
         reversed: true
     }).on("slide", function(e) {
-        $("#tempo-label").html("Tempo: " + getTempo() + " bpm");
+        $("#tempo-label").html("<i class=\"glyphicon glyphicon-time\"></i> : " + getTempo() + " bpm");
         paloData.setTempoInfo();
     }).on("slideStop", function(e) {
-        $("#tempo-label").html("Tempo: " + getTempo() + " bpm");
+        $("#tempo-label").html("<i class=\"glyphicon glyphicon-time\"></i> : " + getTempo() + " bpm");
         paloData.setTempoInfo();
     });
     // Force rendering of the slider's label
@@ -1208,108 +1208,114 @@ function buildUi() {
     var html = "";
 
     // Visualization
-    html += "<div id=\"visualizer-row\" class=\"row\">";
-    html += "<div class=\"col-xs-12\">";
-    html += "<div id=\"visualizer\">";
-    html += "</div>";
-    html += "</div>";
+    html += "<div id=\"visualizer-row\" class=\"container custom-row\">";
+    html += "    <div class=\"row\">";
+    html += "        <div id=\"visualizer\" class=\"col-xs-12\">";
+    html += "        </div>";
+    html += "    </div>";
     html += "</div>";
 
-    html += "<div id=\"controls-row\" class=\"row\">";
+    html += "<div id=\"controls-row\" class=\"container custom-row\">";
+    html += "    <div class=\"row\">";
 
     // Palo switcher + options (resolution and palmas) + play button
-    html += "<div id=\"left-col\" class=\"col-xs-6 col-sm-6 col-md-8 col-lg-8\">";
-    html += "<div class=\"row\">";
+    html += "        <div id=\"left-col\" class=\"col-xs-6 col-sm-6 col-md-8 col-lg-8\">";
+    html += "            <div class=\"row\">";
 
     // Palo switcher + options
-    html += "<div id=\"palo-and-options\" class=\"col-xs-12 col-sm-12 col-md-6 col-lg-6\">";
+    html += "                <div id=\"palo-and-options\" class=\"col-xs-12 col-sm-12 col-md-6 col-lg-6 custom-row\">";
 
     // Palo switcher
-    html += "<select id=\"palo\" class=\"form-control\">";
+    html += "                    <select id=\"palo\" class=\"form-control custom-row\">";
     $.each(window.aCompas.palos, function(paloIndex, paloData) {
-        html += "<option value=\"" + paloData.slug + "\">";
+        html += "                    <option value=\"" + paloData.slug + "\">";
         html += paloData.label;
-        html += "</option>";
+        html += "                    </option>";
     });
-    html += "</select>";
+    html += "                    </select>";
 
     // Options
 
     // Resolution
-    html += "<div>";
-    html += "    <div class=\"btn-group\" role=\"group\">";
-    html += "        <button class=\"resolution resolution-0 btn btn-default btn-sm active\" title=\"Up beats and down beats\"><img src=\"common/images/croche.svg\" class=\"btn-icon\" />";
-    html += "        <button class=\"resolution resolution-1 btn btn-default btn-sm\" title=\"Up beats only\"><img src=\"common/images/noire.svg\" class=\"btn-icon\" />";
-    html += "    </div>";
-    html += "</div>";
+    html += "                    <div>";
+    html += "                        <div class=\"btn-group btn-group-justified custom-row\" role=\"group\">";
+    html += "                            <div class=\"btn-group\" role=\"group\">";
+    html += "                                <button class=\"resolution resolution-0 btn btn-default btn-sm active\" title=\"Up beats and down beats\"><img src=\"common/images/croche.svg\" class=\"btn-icon\" />";
+    html += "                             </div>";
+    html += "                             <div class=\"btn-group\" role=\"group\">";
+    html += "                                <button class=\"resolution resolution-1 btn btn-default btn-sm\" title=\"Up beats only\"><img src=\"common/images/noire.svg\" class=\"btn-icon\" />";
+    html += "                             </div>";
+    html += "                        </div>";
+    html += "                    </div>";
 
     // Instruments
-    html += "<div>";
-    html += "    <div class=\"btn-group\" role=\"group\">";
-    html += "        <button id=\"toggle-instruments\" class=\"btn btn-default btn-lg\" title=\"Toggle instruments\"><i class=\"glyphicon glyphicon-th-list\"></i> Instruments</button>";
-    html += "    </div>";
-    html += "    <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"clara\" title=\"Palma clara\"><img src=\"common/images/clara.svg\" class=\"btn-instrument\" /></button>";
-    html += "    <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"sorda\" title=\"Palma sorda\"><img src=\"common/images/sorda.svg\" class=\"btn-instrument\" /></button>";
-    html += "    <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"cajon\" title=\"Cajón\"><img src=\"common/images/cajon.svg\" class=\"btn-instrument\" /></button>";
-    html += "    <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"udu\" title=\"Udu\"><img src=\"common/images/udu.svg\" class=\"btn-instrument\" /></button>";
-    html += "    <button class=\"toggle-instrument btn btn-default btn-lg btn-circle\" data-instrument=\"click\" title=\"Click\"><img src=\"common/images/click.svg\" class=\"btn-instrument\" /></button>";
-    html += "</div>";
+    html += "                    <div>";
+    html += "                        <button id=\"toggle-instruments\" class=\"btn btn-default btn-lg btn-block custom-row\" title=\"Toggle instruments\"><i class=\"glyphicon glyphicon-th-list\"></i> Instruments</button>";
+    html += "                        <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"clara\" title=\"Palma clara\"><img src=\"common/images/clara.svg\" class=\"btn-instrument\" /></button>";
+    html += "                        <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"sorda\" title=\"Palma sorda\"><img src=\"common/images/sorda.svg\" class=\"btn-instrument\" /></button>";
+    html += "                        <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"cajon\" title=\"Cajón\"><img src=\"common/images/cajon.svg\" class=\"btn-instrument\" /></button>";
+    html += "                        <button class=\"toggle-instrument btn btn-default btn-lg btn-circle active\" data-instrument=\"udu\" title=\"Udu\"><img src=\"common/images/udu.svg\" class=\"btn-instrument\" /></button>";
+    html += "                        <button class=\"toggle-instrument btn btn-default btn-lg btn-circle\" data-instrument=\"click\" title=\"Click\"><img src=\"common/images/click.svg\" class=\"btn-instrument\" /></button>";
+    html += "                    </div>";
 
     // Improvise
-    html += "<div>";
-    html += "    <div class=\"btn-group\" role=\"group\">";
-    html += "        <button id=\"improvise\" class=\"btn btn-default btn-lg active\" title=\"Add some randomness to the rhythmic pattern\"><i class=\"glyphicon glyphicon-random\"></i> Improvise</button>";
-    html += "    </div>";
-    html += "</div>";
+    html += "                    <div>";
+    html += "                        <button id=\"improvise\" class=\"btn btn-default btn-lg btn-block custom-row active\" title=\"Add some randomness to the rhythmic pattern\"><i class=\"glyphicon glyphicon-random\"></i> Improvise</button>";
+    html += "                    </div>";
 
-    html += "</div>"; // End #palo-and-options
+    html += "                </div>"; // End #palo-and-options
 
     // Play button + info zone
-    html += "<div id=\"play-and-info-container\" class=\"col-xs-12 col-sm-12 col-md-6 col-lg-6\">";
-    html += "<div class=\"row\">"
-    html += "<div class=\"col-xs-6 col-sm-6 col-md-12 col-lg-12\">";
+    html += "                <div id=\"play-and-info-container\" class=\"col-xs-12 col-sm-12 col-md-6 col-lg-6 custom-row text-center\">";
+    html += "                    <div class=\"row\">"
+    html += "                        <div class=\"col-xs-6 col-sm-6 col-md-12 col-lg-12\">";
 
     // Play button
-    html += "<button class=\"play\">";
-    html += "<span class=\"glyphicon glyphicon-play\" aria-hidden=\"true\"></span>";
-    html += "</button>";
+    html += "                            <button class=\"play\">";
+    html += "                                <span class=\"glyphicon glyphicon-play\" aria-hidden=\"true\"></span>";
+    html += "                            </button>";
 
-    html += "</div>";
-    html += "<div class=\"col-xs-6 col-sm-6 col-md-12 col-lg-12\">";
+    html += "                        </div>";
+    html += "                        <div class=\"col-xs-6 col-sm-6 col-md-12 col-lg-12\">";
 
     // Info area
-    html += "<div class=\"row\">"
-    html += "<p id=\"info\" class=\"text-danger text-center\"></p>";
-    html += "</div>";
+    html += "                            <div class=\"row\">"
+    html += "                                <p id=\"info\" class=\"text-danger text-center\"></p>";
+    html += "                            </div>";
 
-    html += "</div>";
-    html += "</div>"; // End .row
-    html += "</div>"; // End #play-and-info-container
+    html += "                        </div>";
+    html += "                    </div>"; // End .row
+    html += "                </div>"; // End #play-and-info-container
 
-    html += "</div>"; // End .row
-    html += "</div>"; // End Palo switcher + options (resolution and palmas) + play button
+    html += "            </div>"; // End .row
+    html += "        </div>"; // End Palo switcher + options (resolution and palmas) + play button
 
     // Sliders (tempo and volume)
-    html += "<div id=\"right-col\" class=\"col-xs-6 col-sm-6 col-md-4 col-lg-4\">";
-    html += "<div class=\"row\">";
+    html += "        <div id=\"right-col\" class=\"col-xs-6 col-sm-6 col-md-4 col-lg-4 text-center\">";
+    html += "            <div class=\"row\">";
 
     // Tempo
-    html += "<div id=\"tempo-slider-container\" class=\"col-xs-6\">";
-    html += "<label id=\"tempo-label\" class=\"label label-default\">Tempo:</label>";
-    html += "<div id=\"tempo\">";
-    html += "</div>"
-    html += "</div>";
+    html += "                <div id=\"tempo-slider-container\" class=\"col-xs-6\">";
+    html += "                    <div class=\"row\">";
+    html += "                        <label id=\"tempo-label\"><i class=\"glyphicon glyphicon-time\"></i> : </label>";
+    html += "                    </div>"
+    html += "                    <div id=\"tempo\">";
+    html += "                    </div>";
+    html += "                </div>";
     // Volume
-    html += "<div id=\"volume-slider-container\" class=\"col-xs-6\">";
-    html += "<label id=\"volume-label\" class=\"label label-default\">Volume: " + window.aCompas.masterVolume + " %</label>";
-    html += "<div id=\"volume\">";
-    html += "</div>";
-    html += "</div>";
+    html += "                <div id=\"volume-slider-container\" class=\"col-xs-6\">";
+    html += "                    <div class=\"row\">";
+    html += "                        <label id=\"volume-label\"><i class=\"glyphicon glyphicon-volume-up\"></i> : " + window.aCompas.masterVolume + " %</label>";
+    html += "                    </div>"
+    html += "                    <div id=\"volume\">";
+    html += "                    </div>";
+    html += "                </div>";
 
-    html += "</div>";
-    html += "</div>"; // End #right-col
+    html += "            </div>";
+    html += "        </div>"; // End #right-col
 
-    html += "</div>"; // End .row
+    html += "    </div>"; // End .row
+    html += "</div>"; // End #controls-row
 
     $("#main").html(html);
 
@@ -1342,10 +1348,10 @@ function buildUi() {
         reversed: true,
     }).on("slide", function(e) {
         window.aCompas.masterVolume = e.value;
-        $("#volume-label").html("Volume: " + window.aCompas.masterVolume + " %");
+        $("#volume-label").html("<i class=\"glyphicon glyphicon-volume-up\"></i> : " + window.aCompas.masterVolume + " %");
     }).on("slideStop", function(e) {
         window.aCompas.masterVolume = e.value;
-        $("#volume-label").html("Volume: " + window.aCompas.masterVolume + " %");
+        $("#volume-label").html("<i class=\"glyphicon glyphicon-volume-up\"></i> : " + window.aCompas.masterVolume + " %");
     });
 
     $('.play').on('click', function() {
diff --git a/cordova/www/index.html b/cordova/www/index.html
index 9f40b47..925a99f 100644
--- a/cordova/www/index.html
+++ b/cordova/www/index.html
@@ -26,12 +26,14 @@
     <body>
 
         <div id="wrapper">
-            <header class="container">
-                <h1 id="title">A Compás</h1>
-                <h2 id="baseline"><small>A&nbsp;flamenco&nbsp;metronome&nbsp;mobile&nbsp;app</small></h2>
+            <header>
+                <div class="container custom-row text-center">
+                    <h1 id="title">A Compás</h1>
+                    <h2 id="baseline"><small>A&nbsp;flamenco&nbsp;metronome&nbsp;mobile&nbsp;app</small></h2>
+                </div>
             </header>
 
-            <section id="main" class="container">
+            <section id="main">
             </section>
         </div>
 
diff --git a/web/index.html b/web/index.html
index 58f16ee..d75e7f1 100644
--- a/web/index.html
+++ b/web/index.html
@@ -19,16 +19,18 @@
     <body>
 
         <div id="wrapper">
-            <header class="container">
-                <h1 id="title">A Compás</h1>
-                <h2 id="baseline"><small>A&nbsp;flamenco&nbsp;metronome&nbsp;for&nbsp;the&nbsp;Web</small></h2>
+            <header>
+                <div class="container custom-row text-center">
+                    <h1 id="title">A Compás</h1>
+                    <h2 id="baseline"><small>A&nbsp;flamenco&nbsp;metronome&nbsp;for&nbsp;the&nbsp;Web</small></h2>
+                </div>
             </header>
 
-            <section id="main" class="container">
+            <section id="main">
             </section>
 
             <footer>
-                <div class="container">
+                <div class="container text-center">
                     <p>
                         <a href="https://play.google.com/store/apps/details?id=audio.acompas.app"
                             title="Get A Compás for Android" target="_blank">
